site/ja/blog/new-in-devtools-94/index.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: "layouts/blog-post.njk"
+title: "DevTools の新機能 (Chrome 94)"
+authors:
+  - jecelynyeen
+date: 2021-08-24
+updated: 2021-08-24
+description:
+  "好みの言語での DevTools の利用, 新しい Nest Hub デバイス, 新しい CSS コンテナクエリバッチなど。"
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/js1uKEulNBGj0PoJkqZq.jpg'
```

Comment:
Replace with `image/dPDCek3EhZgLQPGtEG3y0fTn4v82/7mzgwpbt5F3oEUb8d94u.jpg`

Reply:
@chybie I changed the image URL at 91aae57.